### DISCUSSION TITLE
Search should updated the numbers when selecting

### DISCUSSION THREAD

**Discussion by RogerSik** [jonaswinkler/paperless-ng#921] [Feature Requests]
Currently then numbers behind the search will always show the max result.

But it would be nice that the number would get updated for easier searching.

Example:

![14_20-06-22-Dokumente - Paperless-ng und 4 weitere Seiten - Sikorski – Microsoft​ Edge](https://user-images.githubusercontent.com/165814/114758220-02d58880-9d5d-11eb-80fe-a11bb7be4be3.png)

in this constellation there are 0 documents for this correspondence but it will show 5 because of other correspondence.

**Comment by shamoon** [comment]
Dupe #514 #898
